python_cross_site_scripting/flowsn_proplookup/1.py

```
@@ -0,0 +1,256 @@
+# Build ACRIS document image URL
+def generate_document_url(doc_id):
+    return f"https://a836-acris.nyc.gov/DS/DocumentSearch/DocumentImageView?doc_id={doc_id}"
+
+def get_bbl_from_address(street, borough):
+    try:
+        num, name = street.split(' ', 1)
+    except ValueError:
+        return None, None, None, None, None, None, {}
+    headers = {'Ocp-Apim-Subscription-Key': GEOCLIENT_KEY}
+    params = {'houseNumber': num, 'street': name, 'borough': borough}
+    resp = requests.get('https://api.nyc.gov/geoclient/v2/address', params=params, headers=headers)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)